news_home.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>News Homepage</title>
+    <link rel="stylesheet" href="home.css">
+  </head>
+
+  <body>
+    <h1>New News</h1>
+      <ul>
+        <%articles_array.each do |article| %>
```

Comment:
```ruby
articles = {}
articles[:title1] = { :url => "foo.com", :description => "bar" }
```
I would avoid calling it url1 and url2 for keys because then you have to keep track of that just call them url